api/v1alpha1/envoyproxy_metric_types.go

```diff
@@ -23,7 +23,7 @@ type ProxyMetrics struct {
 	// `cluster.<cluster_name>.membership_degraded`，reference  https://github.com/envoyproxy/envoy/issues/9856,
 	// https://github.com/envoyproxy/envoy/issues/14610
 	//
-	Matches []Match `json:"matches,omitempty"`
+	Matches []StatsMatch `json:"matches,omitempty"`
```

Comment:
replacing this with `sringsMatch`, should be backwards compatible from a YAML perspective, right ?

Reply:
Yes, we can. We should align the string match type across APIs. Currently supports exact, suffix, prefix, and Regex. Others can be included if needed in the future.